x/evm/keeper/grpc_query.go

```diff
@@ -300,7 +304,7 @@ func (k Keeper) EstimateGasInternal(c context.Context, req *types.EthCallRequest
 	} else {
 		// Query block gas limit
 		params := ctx.ConsensusParams()
-		if params != nil && params.Block != nil && params.Block.MaxGas > 0 {
+		if params.Block != nil && params.Block.MaxGas > 0 {
```

Comment:
params cant be nil now?

Reply:
Now the `ctx.ConsensusParams()` returns a `ConsensusParams` type (not a pointer). So if it is not defined, will be an empty `ConsensusParams{}` and not nil